core/app/models/spree/variant.rb

```diff
@@ -217,6 +217,15 @@ def price_modifier_amount(options = {})
       end.sum
     end
 
+    def compare_at_price
```

Comment:
@aplegatt please implement this in https://github.com/spree/spree/blob/d2a72bca801d223eec6eb899d146f2586ba44412/core/app/models/concerns/spree/default_price.rb#L11